src/fakeprinter/fakeprinter/middlewares

```diff
@@ -0,0 +1,15 @@
+
+class ThrottlingMiddleware(object):
+    '''
+    delays any request 
+    '''
+
+    def __init__(self, delay_in_seconds, app):
+        self.app = app
+        self.delay = delay_in_seconds
+
+    def __call__(self, environ, start_response):
+        throttle = app.config.get("THROTTLE")
+        if throttle:
+            time.sleep(self.delay_in_seconds)
```

Comment:
self.delay_in_seconds -> self.delay

Reply:
fixed in 497638cc030d6e140d703934e0feb7120e1154b5